assets/src/edit-story/components/header/buttons.js

```diff
@@ -167,24 +170,27 @@ function PreviewButton() {
 }
 
 function Publish() {
```

Comment:
Perhaps it's time to split this into a separate file to avoid getting the file bloated?

Reply:
Fixed in 800b010d8.